ports/plf-queue/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "plf-queue",
+  "version": "2.0.2",
```

```suggestion
  "version-date": "2023-11-26",
```
Since there is no available tag or release in the upstream, we need to use the `version-date` type instead of `version`.